config/usersync.go

```diff
@@ -2,13 +2,13 @@ package config
 
 // UserSync specifies the static global user sync configuration.
 type UserSync struct {
-	Cooperative UserSyncCooperative `mapstructure:"coop_sync"`
-	ExternalURL string              `mapstructure:"external_url"`
-	RedirectURL string              `mapstructure:"redirect_url"`
+	Cooperative    UserSyncCooperative `mapstructure:"coop_sync"`
+	ExternalURL    string              `mapstructure:"external_url"`
+	RedirectURL    string              `mapstructure:"redirect_url"`
+	PriorityGroups [][]string          `mapstructure:"priority_groups"`
```

Comment:
Why are you moving priority groups down a level?

Reply:
The requirements from the issue https://github.com/prebid/prebid-server/issues/2173, calls for this value to be moved up a level from `usersync.coop_sync.pri_groups` --> `usersync.pri_groups`

`Several configurations were placed at the "coop-sync" level that really belong up at the parent cookie-sync level. The proposal is move them around to align better with usage.

cookie-sync.coop-sync.pri // move to the cookie-sync level`